music21/musicxml/test_m21ToXml.py

```diff
@@ -91,6 +91,78 @@ def testSpannersWrite(self):
         p.insert(0.0, sl3)
         self.assertEqual(self.getXml(p).count('<slur '), 6)
 
+    def testSpannerAnchors(self):
```

Comment:
This is the output of this test case rendered by Finale. Notice the music is now shown as Voice 2, which is reasonable to assume from the `<forward>` tags, and isn't great. Can we do the hairpins _after_ the other elements?

<img width="541" alt="finale-voice-2" src="https://user-images.githubusercontent.com/38668450/209451628-94a8a9f2-0fdd-4d38-85e2-bf44d9ad3a03.png">

Reply:
This file actually crashes MuseScore 4, see musescore/MuseScore#15575

Reply:
I've fixed the simple requests here.  The DRY stuff and reordering to avoid weirdly incremented voice numbers are still being worked on.

Reply:
Done.